src/NuGet.Protocol.Catalog/Models/BasePackageDependencyGroup.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class BasePackageDependencyGroup<TDependency> where TDependency : PackageDependency
```

Comment:
Would it make sense to be abstract ?

Reply:
yes, you're right. It should be abstract.

Reply:
Done.